src/transport/raw/ActiveTCPConnectionState.h

```diff
@@ -123,6 +123,10 @@ struct ActiveTCPConnectionState
 
     void Init(Inet::TCPEndPoint * endPoint, const PeerAddress & peerAddr, ReleaseFnType releaseConnection)
     {
+        if (endPoint)
+        {
+            endPoint->Retain();
```

Comment:
I presume this set of Retain()/Release() is basically taking the place of the Retain()/Release() that existed(and now removed) in TCPEndPoint when it was Connecting, right?

Reply:
No, unrelated - this wasn't needed before because effectively we had a dangling pointer with a ref count = 2

We now have proper ref counting, and the `unique_ptr endPointHolder` in TCP.cpp (your other question) will decrement the ref count; we have to up the ref count here because we have a pointer (I have a follow-up PR to make this more automated & cleaner)